src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -446,15 +447,17 @@ class EthService(
     }
   }
 
+  //noinspection ScalaStyle
```

Comment:
Is it the same as scalastyle:off. Why is it needed?

Reply:
I guess the `scalastyle:off` at the top only apply to the two specific styles mentioned but I do not remeber why I introcuced this, so it is [gone now](https://github.com/input-output-hk/mantis/pull/410/commits/63ad75053be3d4047e966b071915f0bb445f4279)!